### PR TITLE
Move config into country specific subfolders

### PR DESCRIPTION
**Pull request by ericboucher**
The ultimate goal is to have all country configs in master to avoid merge conflicts as we start working on more countries simultaneously.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,21 +7,27 @@ This project is the front-end interface for the World Food Programme's [PRISM pr
 ## Functionalities
 
 The new PRISM frontend is built as a static website to minimize cross dependencies and simplify deployments as much as possible. Currently, PRISM frontend provides the ability to:
+
 - Load administrative boundaries as GeoJSON (`src/config/admin_boundaries.json`)
 - Load baseline data as JSON, and link it to administrative boundaries
 - Display WMS layers from Geoserver or Open Data Cube endpoints, with date selection capabilities
 - Display CSV tables in a left side panel
 
+To chose which country to run PRISM for, you can set the environment variable `REACT_APP_COUNTRY`. Currently, available configurations are `mongolia`, `indonesia`. The default is `mongolia`.
```
Comment (Ry-DS) [inline README.md:16]: Not really required, but if its a list
- Item 1
- Item 2

Itll be easy to add to in the future.

File: src/components/MapView/index.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ import {
   loadAvailableDates,
 } from '../../context/serverStateSlice';
 
-import appConfig from '../../config/prism.json';
+import { appConfig } from '../../config/config';
```
Comment (Ry-DS) [inline src/components/MapView/index.tsx:43]: What do you think of using `index.ts` inside the config folder so there isn't a double config?

File: src/config/config.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    appConfig: myanmarConfig,
+    rawLayers: myanmarRawLayers,
+    rawTables: myanmarRawTables,
+  },
+} as const;
+
+type Country = 'indonesia' | 'mongolia' | 'mozambique' | 'myanmar';
+
+const { REACT_APP_COUNTRY: COUNTRY } = process.env;
+const safeCountry = (COUNTRY && Object.keys(configMap).includes(COUNTRY)
```
Comment (Ry-DS) [inline src/config/config.ts:49]: Maybe we could use https://lodash.com/docs/4.17.15#has

File: src/config/config.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+import {
+  mozambiqueConfig,
+  mozambiqueRawLayers,
+  mozambiqueRawTables,
+} from './mozambique';
+
+import { myanmarConfig, myanmarRawLayers, myanmarRawTables } from './myanmar';
+
+const DEFAULT = 'mongolia';
```
Comment (Ry-DS) [inline src/config/config.ts:21]: `const DEFAULT: Country=...`?

**Review by Ry-DS — APPROVED** [review]
Looks good! :+1:

**Comment by ericboucher** [comment]
@hav142 @dubwalla @JorgeMartinezG this is a big change but will make development easier and avoid merge conflicts in the future.

Instead of having a master branch per country, we will use `master` to hold all the configs. The app will default to Mongolia but running another country will be as simple as 

`REACT_APP_COUNTRY=indonesia yarn start` once this PR is merged

**Comment by dubwalla** [comment]
@mwarizmiwafiq @reed1 - please have a look. this change will impact how country level configurations are managed and simplify merging in the future

**Comment by reed1** [comment]
Thanks for notifying Amit, this will indeed makes development easier. If I may, I have several inputs on this:
    
- Country specific subfolders to be added to gitignore and make each country their own git repository to store config. This will make the repository consists of frontend code only. We can also use git-submodule, but I think cloning twice (for frontend and for country config) is simpler.

- Or we can also use dropbox like service to store and serve files, especially for big json and csv. It will help shrinking the frontend repo size.

For now we're still using idn_branch for features like #137. Should I make pull requests for features implemented there? After the features are merged, we can use master and remove idn_dev branch.

**Comment by dubwalla** [comment]
Thanks for the feedback Ridwan. I think the new structure with subfolders per country within the data folder covers this sufficiently. We should continue to have the data folder content available in GitHub if thats what the configuration points to. You can always ignore folders that aren't relevant for your locally. 

I think it's also ok to store large data elsewhere. S3 would probably be the preferred solution. @ericboucher - thoughts?

**Comment by dubwalla** [comment]
> For now we're still using idn_branch for features like #137. Should I make pull requests for features implemented there? After the features are merged, we can use master and remove idn_dev branch.

Yes, you should be creating PRs for your work as a draft PR if still in progress, or as a PR if ready for review.

**Comment by ericboucher** [comment]
@reed1 thanks for your input. Yes, please transition all future PRs to point towards the master branch.

I think having one repo with all the configs will be useful to let developers check that their features are working in other countries as well.

If some data files are/get too big, we will move them to S3 like we are doing for some of the admin boundaries.

Let me know if you have any other questions :)

**Pull request merged**